src/west/main.py

```diff
@@ -48,15 +55,14 @@
 # Built-in commands in this West. For compatibility with monorepo
 # installations of West within the Zephyr tree, we only expose the
 # project commands if this is a multirepo installation.
```

Comment:
Mind updating this comment while you are here?

Reply:
Will do

Reply:
updated